### ISSUE TITLE
calling delegateEvents on a view that includes a behavior with a ui binding breaks

### ISSUE THREAD

**Issue by jonknapp** [marionettejs/backbone.marionette#2144]
I ran into a possible bug when trying to call `delegateEvents` after the `ui` hash has been modified for a view's `Behavior`.  It appears that the `Marionette.normalizeUIKeys` method is calling the `Marionette.normalizeUIString` whose purpose is to return the jQuery selector.  This works fine before the view is rendered, however after `ui` elements start to point to jQuery objects, we can no longer call `delegateEvents` again.

It will try to create a `_behaviorsEvents` hash that contains items like:

```
{
  ...
  "keypress [object Object] ": function bound(),
  ...
}
```

`[object Object]` is an invalid selector, which then throws an error in jQuery once the event is caught.

Issue https://github.com/marionettejs/backbone.marionette/issues/1408 seems to relate to it, though on the `View` side.  I haven't dived as deep on that side yet, but it looks like there were steps to try and work with a cached version of the `ui` hash so there was access to the original selectors: https://github.com/marionettejs/backbone.marionette/pull/1731

**Comment by samccone** [comment]
This has been merged and is going out with the 2.4.0 release

**Comment by jasonLaster** [comment]
Great work @jonknapp!
